doc/governance.rst

```diff
@@ -86,6 +86,17 @@ For this, they can operate the scikit-learn accounts on various social networks
 and produce materials. They also have the required rights to our blog
 repository and other relevant accounts and platforms.
 
+Documentation team
+~~~~~~~~~~~~~~~~~~
+
+Members of the documentation team engage with the documentation of the project
+among other things. They might also be involved in other aspects of the
+project, but their reviews on documentation contributions of others are
+considered authoritative, and can merge such contributions.
```

Comment:
```suggestion
Members of the documentation team engage with the documentation of the project
among other things. They might also be involved in other aspects of the
project, but their reviews on documentation contributions are
considered authoritative.
```

Reply:
I prefer to be explicit here. The governance is a contract, and we should move it towards a more explicit text rather than implicit assumptions.